README.rst

```diff
@@ -71,6 +71,10 @@ format:
    ddf.to_parquet("path/to/dir/")
    ddf = dask_geopandas.read_parquet("path/to/dir/")
 
+.. note::
+
+   Writing to Parquet files requires installing the ``pyarrow`` library, e.g. ``pip install pyarrow``.
```

Comment:
```suggestion
   Writing to Parquet files requires installing the ``pyarrow`` library, e.g.
   ``conda install pyarrow`` or ``pip install pyarrow``.
```
GeoPandas is much safer to install via conda, so we're trying to point users towards conda if we can.